src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -443,7 +451,7 @@ void LiveFunctionsDataView::AddFunction(uint64_t function_id,
 void LiveFunctionsDataView::OnDataChanged() {
   functions_.clear();
   indices_.clear();
-  app_->ShowHistogram({}, "");
+  app_->ShowHistogram({}, "", orbit_grpc_protos::kInvalidFunctionId);
```

Comment:
I would rather set this as the default parameter.

Reply:
The style guide bans default arguments on virtual functions.